### ISSUE TITLE
Release

### ISSUE THREAD

**Issue by kohend** [Aorimn/dislocker#314]
There hasn't been a new release for almost 3 years, is a new release planned anytime soon? I don't like using my own compiled version for a long time.

**Comment by haobinnan** [comment]
@Aorimn

**Comment by superbonaci** [comment]
https://github.com/Aorimn/dislocker/issues/345

**Comment by neuhaus** [comment]
I agree that we need a new release, as it stands i have to compile dislocker myself from the git repository every time i install a new Linux distribution, even cutting edge ones like Ubuntu 25.10 or Fedora 43 because only the git version of dislocker can mount a Windows 11 Bitlocker partition.
I don't understand why @Aorimn closes this issue as "completed".

**Comment by Aorimn** [comment]
@neuhaus did you look at #345 ? I may have missed the reason indeed, that could be confusing. Not sure how to change that now though

**Comment by superbonaci** [comment]
@neuhaus the issue says **completed** but that's the way Github works by default, likely means **not planned** or just **closed**. The only important thing in Github is if the issue is open or closed, the rest are tags that most of the time are not filled in correctly.

Distros have their own maintainers, which may need to patch and adjust dislocker for compatibility with the version you want, ubuntu 25.10 or the future 26.04 -  some changes may be hold back due to compatibility with other libraries.

- Ubuntu -> https://launchpad.net/ubuntu/+source/dislocker
- Fedora -> https://packages.fedoraproject.org/pkgs/dislocker/fuse-dislocker/ for stable release or https://copr.fedorainfracloud.org/coprs/azrdev/dislocker for COPR

If you want to use always dislocker-git you should use rolling release distros instead of point release distros. Some point release distros offer third party repos like COPR on Fedora but are not well maintained. If you want rolling release, use Arch, which has AUR and has packages that are not tied to a specific commit - those ending in `-git`. Other distros like Gentoo or NixOS pin the packages at specific git commit so it's not what you want. You want this: https://aur.archlinux.org/packages/dislocker-git

**Comment by neuhaus** [comment]
I was looking at the tags at https://github.com/Aorimn/dislocker/tags and the latest one is from 2020.
My expectation is that if you add a new tag then the distributions will relase updated packages and you don‘t need to add an extra repository which is a hassle and can be a problem.

**Comment by Aorimn** [comment]
As previously explained, I won't tag a new version until the tests are fixed.

**Comment by superbonaci** [comment]
@neuhaus if you want to download a "release" to make the build reproducible you can get the zip of any commit, maintainers already know that, for example the latest right now:
https://github.com/Aorimn/dislocker/archive/8b2aea09d431bd5497ae223c141ebaee7bdd481f.zip

**Comment by neuhaus** [comment]
> [@neuhaus](https://github.com/neuhaus) if you want to download a "release" to make the build reproducible you can get the zip of any commit, maintainers already know that, for example the latest right now: https://github.com/Aorimn/dislocker/archive/8b2aea09d431bd5497ae223c141ebaee7bdd481f.zip

I don't agree that maintainers are aware of important features missing since the last release and switching to git commits. Case in point: Both Ubuntu 25.10 and Fedora 43 ship with a version of dislocker that cannot decrypt current Windows 11 bitlocker partitions, yet the git version can do it. Apparently this important feature was added since the last release.

**Comment by neuhaus** [comment]
> As previously explained, I won't tag a new version until the tests are fixed.

I understand. What do you think about tagging a version as alpha or beta version?

**Comment by superbonaci** [comment]
@neuhaus probably dislocker is not used that much in the business world so it's not kept up to date with every new commit. You have to understand that people who needs use Bitlocker encrypted partitions, that's because they already use Windows Professional or Windows Server.

Basically releases are nothing but a git commit in a zip file like I pasted before. Projects who use extensively releases is because they release **precompiled** binaries for several platforms and they are the original developers, it would be like if Microsoft released dislocker themselves. Check here for example: https://github.com/powershell/powershell/releases
